tailwind.config.js

```diff
@@ -32,6 +32,17 @@ export default {
 
     theme: {
         extend: {
+            typography: {
```

Comment:
You are overwriting prose classes with our adaptive font? Much better than creating a helper class like it was before

Reply:
yeah much cleaner, no need for the text-markdown anymore!